app/src/main/AndroidManifest.xml

```diff
@@ -32,25 +41,26 @@
         <meta-data
             android:name="com.google.android.geo.API_KEY"
             android:value="@string/google_maps_key" />
+        <meta-data
+            android:name="preloaded_fonts"
+            android:resource="@array/preloaded_fonts" />
 
         <activity
             android:name=".maps.MapsActivity"
             android:exported="true"
             android:label="AIzaSyBUM-C1dj9pPVRF1pUOoiXhmZ_xCwNj2Bk" />
         <activity
             android:name=".auth.sign_in"
-            android:exported="true">
-
-        </activity>
+            android:exported="true"></activity>
         <activity android:name=".MainActivity" />
-        <activity android:name=".auth.sign_up"
+        <activity
+            android:name=".auth.sign_up"
             android:exported="true">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
 
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
-
         </activity>
```

Comment:
splash screen should come first right ? remove intent-filter

Reply:
yeah  this intent filter is of no use.i'll remove it

Reply:
fine